queries/python/highlights.scm

```diff
@@ -0,0 +1,150 @@
+;; From tree-sitter-python licensed under MIT License
+; Copyright (c) 2016 Max Brunsfeld
+
+; Identifier naming conventions
+
+;((identifier) @constructor
+ ;(#match? @constructor "^[A-Z]"))
+
+;((identifier) @constant
+ ;(#match? @constant "^[A-Z][A-Z_]*$"))
+
+; Function calls
+
+(decorator) @function
+
+(call
+  function: (attribute attribute: (identifier) @function.method))
+(call
+  function: (identifier) @function)
+
+;; Builtin functions
+
+;((call
+  ;function: (identifier) @function.builtin)
+ ;(#match?
+   ;@function.builtin
+   ;"^(abs|all|any|ascii|bin|bool|breakpoint|bytearray|bytes|callable|chr|classmethod|compile|complex|delattr|dict|dir|divmod|enumerate|eval|exec|filter|float|format|frozenset|getattr|globals|hasattr|hash|help|hex|id|input|int|isinstance|issubclass|iter|len|list|locals|map|max|memoryview|min|next|object|oct|open|ord|pow|print|property|range|repr|reversed|round|set|setattr|slice|sorted|staticmethod|str|sum|super|tuple|type|vars|zip|__import__)$"))
+
+;; Function definitions
+
+(function_definition
+  name: (identifier) @function)
+
+(identifier) @variable
+(attribute attribute: (identifier) @property)
+(type (identifier) @type)
+(parameters
+  (identifier) @parameter) 
+
+; Literals
+
+(none) @constant.builtin
+(true) @constant.builtin
+(false) @constant.builtin
```

Comment:
why is there not `self` here ? I know it is actually not a constant but having it highlighted could be some "recommandation" to use it.